app/filtering.js

```diff
@@ -364,6 +364,17 @@ function registerForHeadersReceived (session, partition) {
       muonCb({ cancel: true })
       return
     }
+
+    let parsedTargetUrl = urlParse(details.url || '')
+    let parsedFirstPartyUrl = urlParse(firstPartyUrl)
+    let trackingHeaders = ['Strict-Transport-Security', 'Expect-CT', 'Public-Key-Pins']
+    if (isThirdPartyHost(parsedFirstPartyUrl.hostname, parsedTargetUrl.hostname)) {
+      trackingHeaders.forEach(function (header) {
+        delete details.responseHeaders[header]
+        delete details.responseHeaders[header.toLowerCase()]
```

Comment:
this line shouldn't be needed because `details.responseHeaders` capitalization is already normalized

Reply:
I checked yesterday, and I did see some headers in lowercase. `Strict-Transport-Security` and `strict-transport-security`

Reply:
thanks for checking - i wonder if we should also be checking `cookie` and `referer` in lowercase then

this seems potentially problematic because HTTP headers are case-insensitive according to the RFC. so someone could create a header called `sTriCT-TransPort-SecUrITY` and the browser would still process it (in theory)